profile/latest/advisor/management/advisor/models.go

```diff
@@ -0,0 +1,40 @@
+package advisor
+
+import (
+	 original "github.com/Azure/azure-sdk-for-go/service/advisor/management/2017-04-19/advisor"
```

Comment:
qq, for `compute`, are you generating `compute` and `containerservice` separately? I tink this would be a good moment to merge `containerservice` and `disks` back to `compute`